includes/blocks/header/render.php

```diff
@@ -79,37 +81,57 @@ function benenson_render_header_block( array $attributes = [] ) {
 				esc_url( wp_get_attachment_url( $attributes['featuredVideoId'] ) )
 			);
 		}
-
-		print '<div class="container"><div class="hero-content">';
-
-		if ( $attributes['title'] ) {
-			printf( '<h1 class="page-heroTitle"><span>%s</span></h1>', wp_kses_post( $attributes['title'] ) );
-		}
-
-		if ( $attributes['content'] ) {
-			printf( '<p class="page-heroContent">%s</p>', wp_kses_post( $attributes['content'] ) );
-		}
-
-		if ( $attributes['ctaText'] ) {
-			if ( $attributes['ctaLink'] && ! $attributes['embed'] ) {
-				printf(
-					'<div class="page-heroCta"><a class="btn" href="%s">%s</a></div>',
-					esc_url( $attributes['ctaLink'] ),
-					esc_html( wp_strip_all_tags( $attributes['ctaText'] ) )
-				);
-			} elseif ( $attributes['embed'] ) {
-				printf(
-					'<div class="page-heroCta"><a class="btn" href="%s" data-modal-embed="%s"><i class="play-icon">%s</i>%s</a></div>',
-					esc_url( $attributes['ctaLink'] ),
-					esc_url( $attributes['embed'] ),
-					esc_html__( 'Play video', 'benenson' ),
-					esc_html( $attributes['ctaText'] )
-				);
+		?>
+		<div class="container">
+			<div class="hero-content">
+			<?php
+			if ( $attributes['title'] ) {
+				printf( '<h1 class="page-heroTitle"><span>%s</span></h1>', wp_kses_post( $attributes['title'] ) );
 			}
-		}
-
-		print '</div></div></section>';
 
+			if ( $attributes['content'] ) {
+				printf( '<p class="page-heroContent">%s</p>', wp_kses_post( $attributes['content'] ) );
+			}
+			?>
+
+				<div class="page-heroCta">
```

Comment:
Because this now always exists, if the condition on line 106 is true, it will result in `div.page-heroCta > div.page-heroCta`. Is this intentional?